blueprints/serializer-test/qunit-rfc-232-files/tests/unit/__path__/__test__.js

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { run } from '@ember/runloop';
+
+module('<%= friendlyTestDescription %>', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let store = this.owner.lookup('service:store');
+    let serializer = this.owner.factoryFor('serializer:<%= dasherizedModuleName %>').create({ store });
```

Comment:
I think this should be `.lookup` (not `factoryFor().create()`) because serializers are singleton...

Reply:
you need the store initialized on the serializer though, which lookup doesn't automatically do
is that something that should be addressed upstream?

Reply:
Haha, this seems sorta bonkers. I submitted https://github.com/emberjs/data/pull/5300 to simplify greatly 😺

Reply:
@alexander-alvarez - Even without #5300, we should do `store.serializerFor(<%= dasherizedModelName %>)` here as that is the public API for getting a given things serializer. There are a few fallbacks that Ember Data handles nicely for you (e.g. use application if you don't have a model specific one) and we should go through the same common API.

Reply:
understood :+1: